lib/src/common/variables.js

```diff
@@ -37,7 +41,9 @@ export const globalTokens = {
   hal_blue_s_35: "#0067b3",
   hal_blue_d_20: "#003c66",
   color_blue_200: "#cceaff",
-  color_blue_500: "#33aaff",
+
+  color_blue_500: "#33AAFF",
+  color_blue_600: "#0095FF",
   color_blue_50: "#f5fbff",
```

Comment:
Please keep `#33aaff` in lowercase.
`#0095ff` already exists in `hal_blue_l_50`.

Reply:
done